acoustid/future/fpindex/client.py

```
@@ -0,0 +1,282 @@
+        """
+        self.base_url = base_url.rstrip("/") + "/"
+        self.timeout = timeout
+        self.session = aiohttp.ClientSession()
+
+    async def __aenter__(self):
+        pass
+
+    async def __aexit__(self, exc_type, exc_val, exc_tb):
+        await self.close()
+
+    async def close(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `close`

Add return type annotation: `None`

(ANN201)